Packs/AzureStorage/Integrations/AzureStorage/AzureStorage.py

```diff
@@ -260,24 +265,26 @@ def storage_blob_service_properties_set_request(self, args: Dict) -> Dict:
 
         return self.ms_client.http_request(
             method='PUT',
-            url_suffix=f'/resourceGroups/{self.resource_group_name}/providers/Microsoft.Storage/storageAccounts/'
-                       f'{args["account_name"]}/blobServices/default',
-            params={
-                'api-version': API_VERSION,
-            },
+            full_url=f'https://management.azure.com/subscriptions/{subscription_id}/resourceGroups/{resource_group_name}/providers/Microsoft.Storage/\
+storageAccounts/{account_name}/blobServices/default?api-version={API_VERSION}',
             json_data={'properties': properties}
         )
 
-    def storage_blob_containers_create_update_request(self, args, method):
+    def storage_blob_containers_create_update_request(self, subscription_id: str | None,
+                                                      resource_group_name: str | None, args: Dict, method: str) -> Dict:
         """
                 Send the user arguments for the create blob container in the request body to the API.
             Args:
+                subscription_id: The subscription id.
+                resource_group_name: The resource group name.
                 args: The user arguments.
                 method: The method for the http request.
 
             Returns:
                 The json response from the API call.
         """
+        container_name = args.get('container_name', '')
```

Comment:
sme here about account_name and container_name.

Reply:
This was already like that in the URL itself in a more risky way, and I just changed it from args['account'] to args.get('account').

I could extract the value in the command function and pass it to the request func, but im not sure thats better, because any way i need to pass args, so why extract it in a different function theat dos noting with the value except passing it forword?

Reply:
Ok, I agree.